ui/src/pages/frontsite/Documents/DocumentsDetails/LoanRequestForm/LoanRequestForm.js

```diff
@@ -14,6 +14,7 @@ export default class LoanRequestForm extends Component {
     this.state = {
       requestEndDate: '',
       deliveryMethod: '',
+      error_message: '',
```

Comment:
```suggestion
      errorMessage: '',
```
I think prettier will complain here, in general javascript is more camelcase